cobbler/utils.py

```diff
@@ -1504,6 +1504,32 @@ def is_remote_file(file) -> bool:
         return False
 
 
+def command_existing(cmd: str, expected_return_code: Optional[int]) -> bool:
```

Comment:
What do you think about this snippet from https://stackoverflow.com/a/28909933 ?

```python
import shutil
def cmd_exists(cmd):
    return shutil.which(cmd) is not None
```

Reply:
Oh damn. I even used this in the past somewhere else. Yes of course I change that!

Reply:
Fixed via 0026628